tst/testinstall/opers/IsSolvableGroup.tst

```diff
@@ -53,4 +59,13 @@ true
 gap> G := F/[ a^2, b^2, c^2, d^2, (a*b)^3, (b*c)^3, (c*d)^3, (a*c)^2, (a*d)^2, (b*d)^2 ];;
 gap> not IsSolvable(G) and HasIsAbelian(G) and not IsAbelian(G);
 true
+gap> G := F/[ a^2, b^2, c^2, d^2, (a*b)^3, (b*c)^3, (c*d)^3, (a*c)^2, (a*d)^2, (b*d)^2 ];; Size(G);;
+gap> not IsSolvable(G) and HasIsAbelian(G) and not IsAbelian(G);
+true
+gap> F := FreeGroup("a", "x");; a := F.1;; x := F.2;;
+gap> G := F/[x^2*a^8, a^16, x*a*x^(-1)*a];; Size(G);;
+gap> not HasIsSolvableGroup(G) and IsSolvableGroup(G) and HasIsPGroup(G) and IsPGroup(G) and HasPrimePGroup(G) and HasIsNilpotentGroup(G) and IsNilpotentGroup(G);
+true
```

Comment:
We have by now removed most of "tests" like this, they are simply far too fragile. I see why you did it, but I think we need a better system to test individual methods.

Reply:
Rewrote the tests.